src/components/modal/modal.js

```diff
@@ -509,8 +511,18 @@ export default {
     },
     hide (trigger) {
       if (!this.is_visible) {
-        return
+        if (this.is_show_delayed) {
+          // If another modal is started to hide but not hidden yet,
+          // and yet another modal is about to open, wait for it to
+          // open.
+          this.$root.$once('bv::modal::shown', this.doHide)
```

Comment:
This code causes the `trigger` information (if any present) to be lost.

Reply:
Thanks. I will change It as follows:
```suggestion
          this.$root.$once('bv::modal::shown', () => {
            this.doHide(trigger)
          })
```